ecommerce/front/trackers.es6

```diff
@@ -60,18 +60,34 @@ class GATracker extends Tracker {
  * Reliable loader of Google Analytics scripts.
  *
  * Google Analytics scripts (GA) are loading by the Google tag manager (GTM).
- * If GA is still not loaded, the function will delay execution until it is loaded
- * with GA command queue.
- *
- * See this for details about a command queue:
- * http://bit.ly/rf-ga-command-queue
+ * If GA is still not loaded, the function will delay execution until it is loaded.
  */
-function loadGaTransport() {
-  if (!window['ga']) {
-    window.ga = (...args) => {
-      (ga.q = ga.q || []).push(args);
-    };
-    ga.l = 1*new Date();
+function loadGaTransport(onLoadName) {
+  var state = {
+    loaded: false,
```

Comment:
Seems we can move this logic at some `AsyncGaTransport` class.
But it's optional thought